src/arch-riscv64-syscalls.c

```diff
@@ -0,0 +1,539 @@
+#include <string.h>
```

Comment:
Do we want some sort of copyright header here?  If nothing else perhaps the standard LGPL boilerplate?